src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
```

Comment:
`busNumber` is going to be fixed for any given raspi-i2c instance, and allowing it as an option to this method opens us to the possibility that this instance will stomp all over another instance. Let's remove it as an argument and leave this signature the way it was.